state-chain/pallets/cf-witnesser/src/lib.rs

```diff
@@ -479,6 +479,25 @@ impl<T: Config> Pallet<T> {
 		});
 		CallHashExecuted::<T>::insert(witnessed_at_epoch, call_hash, ());
 	}
+
+	pub fn count_votes(callhash: CallHash) -> Option<Vec<(<T as Chainflip>::ValidatorId, bool)>> {
+		let current_epoch = T::EpochInfo::epoch_index();
+		let votes: BitVec<u8, Msb0> =
+			BitVec::from_vec(Votes::<T>::get(current_epoch, callhash).unwrap_or(vec![]));
+		if votes.is_empty() {
+			return None
+		}
```

Comment:
nit: I think this is equivalent and more idiomatic:

```suggestion
			BitVec::from_vec(Votes::<T>::get(current_epoch, callhash)?);
```

A lot of these variables can also be inlined.